kaldb/src/main/java/com/slack/kaldb/server/ArmeriaService.java

```diff
@@ -146,9 +146,9 @@ private LoggingServiceBuilder getLoggingServiceBuilder() {
 
   @Override
   protected void startUp() throws Exception {
+    LOG.info("Starting {} on port {}", serviceName, serverPort);
```

Comment:
is it valuable to keep both starting/started log entries?

Reply:
I think it makes troubleshooting failed services a bit easier.

When we provide a list of services to the service manager, they all attempt to start in parallel. When we have a failed service that terminates the JVM it can difficult to understand which services haven't begun to start, which ones are in the middle of their startup, and which ones have completed.